src/Program/CreateProgramRequest.php

```diff
@@ -27,37 +28,53 @@ public function __construct(
         string $url,
         string $active
     ) {
-        $this->organization =$organization;
+        $this->organization = $organization;
         $this->uniqueId = $uniqueId;
-        $this->name =$name;
-        $this->point =$point;
-        $this->phone =$phone;
-        $this->url =$url;
-        $this->active =$active;
+        $this->name = $name;
+        $this->point = $point;
+        $this->phone = $phone;
+        $this->url = $url;
+        $this->active = $active;
     }
 
+    /**
+     * @return int
+     */
+    public function getEnableEmailLogin(): int
+    {
+        return $this->enable_email_login ?? 0;
+    }
+
+    /**
+     * Int (1/0) or Bool true/false
+     * @param mixed $enable_email_login
+     */
+    public function setEnableEmailLogin($enable_email_login): void
```

Comment:
does the :void return type work on PHP 7?

Reply:
:void is in a bunch of other methods so Im assuming the services using this are fine